EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
there certainly should be if there isn't... at least it's an edge case that is possible. a peptide sequence can be shared between multiple proteins, and parsimony won't always be able to select one. if an ambiguous PSM includes one of those, there could be 2+ peptides with 3+ proteins